packages/snapshots/example.js

```diff
@@ -0,0 +1,25 @@
+const fs = require("fs");
+
+const c = require("./differ");
```

Comment:
could we rename these to have more declarative names

Reply:
sorry, shouldn't have included this file